src/components/ErrorPage/ErrorPage.tsx

```diff
@@ -0,0 +1,34 @@
+import "./ErrorPage.scss";
+
+export interface ErrorPageProps {
+  errorMessage: string;
+  originURL: string;
+}
+
+function ErrorPage({errorMessage, originURL}: ErrorPageProps) {
+  return (
+    <>
+      <section className="error-page">
+        <span>{errorMessage}</span>
+        <button
+          id="home-button"
+          onClick={() => {
+            window.location.href = "/";
+          }}
+        >
+          Home
+        </button>
+        <button
+          id="back-button"
+          onClick={() => {
+            window.location.href = originURL;
```

Comment:
You can extract the click handler into a helper method. What I like is to write it like

```
const redirect = (newURL: string) => () => { history.push(newURL) }
```

because it'll enable you to write `onClick={redirect('/')}` :)